spec/schema.json

```diff
@@ -65,6 +65,56 @@
         }
       }
     },
+    "/transactions/{transaction_id}/overseas-entity/{overseas_entity_id}": {
+      "put": {
+        "tags": [
+          "Overseas Entity"
```

Comment:
Wondering if this tag is correct (it matches the one for create) or should it be unique?

Reply:
It can be used to aggregate operation/info, so it does not need to be unique. For more info [here](https://swagger.io/docs/specification/grouping-operations-with-tags/)

Reply:
👍 thanks.